nav2_rotation_shim_controller/src/nav2_rotation_shim_controller.cpp

```diff
@@ -201,6 +242,16 @@ geometry_msgs::msg::PoseStamped RotationShimController::getSampledPathPt()
             "passing off to primary controller plugin.", forward_sampling_distance_));
 }
 
+geometry_msgs::msg::PoseStamped RotationShimController::getSampledPathGoal()
+{
+  if (current_path_.poses.size() == 0) {
+    throw nav2_core::ControllerException(
```

Comment:
I think an invalid path exception is OK here https://github.com/ros-navigation/navigation2/blob/main/nav2_msgs/action/FollowPath.action#L15